wear/src/main/java/io/homeassistant/companion/android/tiles/TemplateTile.kt

```diff
@@ -69,11 +71,12 @@ class TemplateTile : TileService() {
 
             val template = integrationUseCase.getTemplateTileContent()
             val renderedText = try {
-                integrationUseCase.renderTemplate(template, mapOf()) ?: getString(
-                    commonR.string.template_error
-                )
+                integrationUseCase.renderTemplate(template, mapOf()).toString()
             } catch (e: Exception) {
-                getString(commonR.string.template_render_error)
+                Log.i("TemplateTile", "Exception while rendering template", e)
```

Comment:
Probably should be tagged as an error instead of info 😄

Reply:
It should, thanks!